<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Image_Window</class>
 <widget class="QMainWindow" name="Image_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像处理</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>491</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>输入图片</string>
     </property>
     <widget class="QGraphicsView" name="input_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>参数设置</string>
     </property>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>451</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>保存路径</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>431</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="dir_textLabel_2">
          <property name="text">
           <string>路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputDir_textLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="open_outputDir_button">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>方式</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,3">
         <item>
          <widget class="QRadioButton" name="scale_radioButton">
           <property name="text">
            <string>放大倍数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scale_comboBox">
           <item>
            <property name="text">
             <string>x2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x4</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
         <item>
          <widget class="QRadioButton" name="diy_radioButton">
           <property name="text">
            <string>自定义</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,0">
             <item>
              <widget class="QLabel" name="diy_width_text">
               <property name="text">
                <string>宽：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="diy_width_lineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,0">
             <item>
              <widget class="QLabel" name="diy_height_text">
               <property name="text">
                <string>高：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="diy_height_lineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>模型</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QRadioButton" name="auto_radioButton">
         <property name="text">
          <string>自动</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QRadioButton" name="manual_radioButton">
           <property name="text">
            <string>手动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modelSelected_comboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>191</width>
        <height>55</height>
       </rect>
      </property>
      <property name="title">
       <string>场景</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QComboBox" name="scene_comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>181</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>图片信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="inputInfo_groupBox">
        <property name="title">
         <string>输入图片信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
           <item>
            <widget class="QLabel" name="input_height_text">
             <property name="text">
              <string>高：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="input_height">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
           <item>
            <widget class="QLabel" name="input_width_text">
             <property name="text">
              <string>宽：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="input_width">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="outputInfo_groupBox">
        <property name="title">
         <string>输出图片信息</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
           <item>
            <widget class="QLabel" name="output_height_text">
             <property name="text">
              <string>高：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="output_height">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
           <item>
            <widget class="QLabel" name="output_width_text">
             <property name="text">
              <string>宽：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="output_width">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>471</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>输出图片</string>
     </property>
     <widget class="QGraphicsView" name="output_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>1021</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>9</y>
       <width>971</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>其他功能</string>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>19</y>
        <width>441</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>批量操作</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="dir_textLabel">
          <property name="text">
           <string>路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputDir_textLine"/>
        </item>
        <item>
         <widget class="QPushButton" name="open_inputDir_button">
          <property name="text">
           <string>浏览</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="compare_pushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>30</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>对比效果</string>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>671</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="dealingStatus_laebl">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>就绪</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_pushButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_pushButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>120</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionupdate"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="about_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionopen_directory">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionupdate">
   <property name="text">
    <string>更新模型</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
